docs/src/developer/devel-setup.md

```diff
@@ -3,6 +3,62 @@
 Some steps described here overlap the ones described in the
 [installation instructions](../admin/installation.md).
 
+## Using Docker
+
+Inside the `docker` folder you can find all the Dockerfile and docker-compose
+files for several enviroments. To run the development setup, copy the
+`.env.example` to `.env` and update it to the values of your environment.
+
```

Comment:
When running with Docker, there are a few values that will be different from the `.env.example` files that you'll want to change. They are as follows:

In the root `.env`, use these values
```
DB_HOST=db
...
API_BASE_URL="http://phpreport-api:8555"
```
In the `frontend/.env`, use the following:
```
VITE_OIDC_REDIRECT_URL='http://0.0.0.0:5173/web/v2/'
VITE_API_BASE='http://0.0.0.0:8555'
```